solidity/ecdsa/contracts/WalletRegistryGovernance.sol

```diff
@@ -172,6 +175,14 @@ contract WalletRegistryGovernance is Ownable {
     );
     event ReimbursementPoolUpdated(address reimbursementPool);
 
+    event NotifyOperatorInactivityGasOffsetUpdateStarted(
+        uint256 notifyOperatorInactivityGasOffset,
+        uint256 timestamp
+    );
+    event NotifyOperatorInactivityGasOffsetUpdated(
+        uint256 notifyOperatorInactivityGasOffset
+    );
```

Comment:
Can we place them above `ReimbursementPoolUpdateStarted`, right after events for DKG approval gas offset?

Reply:
Sure